lewis/devices/__init__.py

```diff
@@ -139,18 +139,16 @@ def _get_state_handlers(self):
             "_get_state_handlers must be implemented in a StateMachineDevice."
         )
 
-    def _get_initial_state(self):
+    def _get_initial_state(self) -> State:
         """
         Implement this method to return the initial state of the internal state machine.
         The default implementation raises a ``NotImplementedError``.
 
         :return: The initial state of the state machine.
         """
-        raise NotImplementedError(
-            "_get_initial_state must be implemented in a StateMachineDevice."
-        )
+        raise NotImplementedError("_get_initial_state must be implemented in a StateMachineDevice.")
 
-    def _get_transition_handlers(self):
+    def _get_transition_handlers(self) -> dict[tuple[State, State], Callable]:
```

Comment:
could be nice to specify what the signature of the callable is expected to be?

I think it's `Callable[[], bool]`

but not critical